vertical-pod-autoscaler/docs/api.md

```diff
@@ -0,0 +1,399 @@
+# API Reference
+
+## Packages
+- [autoscaling.k8s.io/v1](#autoscalingk8siov1)
+
+
+## autoscaling.k8s.io/v1
+
+Package v1 contains definitions of Vertical Pod Autoscaler related objects.
+
+Package v1 contains definitions of Vertical Pod Autoscaler related objects.
```

Comment:
can we do something about those duplicated lines?

Reply:
The first one comes from here: https://github.com/kubernetes/autoscaler/blob/7df4f842a789e0b28f6162aa8d61a6743dab5590/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/doc.go#L19

The second one comes from here: https://github.com/kubernetes/autoscaler/blob/7df4f842a789e0b28f6162aa8d61a6743dab5590/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L17


Looking at other projects and their docs, I think it's fine to remove the comment from types.go, what do you think?

Reply:
Agree. can you you remove this in this PR and regenerate the API docs?

Reply:
Fixed in 391174612b316451e1147ef37df96a6b5eccbb3f